_delphi_utils_python/delphi_utils/runner.py

```diff
@@ -72,9 +72,13 @@ def run_indicator_pipeline(indicator_fn:  Callable[[Params], None],
     start = time.time()
     while time.time()-start < timer:
         if not t1.is_alive():
+            logger.info("Completed flash step",
+                    elapsed_time_in_seconds = round(time.time() - start, 2))
             break
         time.sleep(10)
```

Comment:
```suggestion
        time.sleep(1)
```
we should be able to bring this sleep time down without creating a busy wait, and itll give tighter timing measurements.